SoftLayer/managers/account.py

```diff
@@ -219,6 +220,28 @@ def get_billing_item(self, identifier, mask=None):
 
         return self.client.call('Billing_Item', 'getObject', id=identifier, mask=mask)
 
+    def get_billing_item_from_invoice(self, identifier):
+        """Gets details about a billing item of a billing invoice item
+
+        :param int identifier: Billing_Invoice_Item id
+        :return: Billing_Item
+        """
+        return self.client.call('Billing_Invoice_Item', 'getBillingItem', id=identifier)
```

Comment:
Should be using the same mask as `get_billing_item`, specifically `,order[id,userRecord[id,email,displayName,userStatus]]]` is missing.

Reply:
Updated